src/Evolution/Initialization/Evolution.hpp

```diff
@@ -29,117 +29,126 @@
+namespace Initialization {
+
+namespace detail {
+inline Time initial_time(const bool time_runs_forward,
+                         const double initial_time_value,
+                         const double initial_slab_size) {
+  const Slab initial_slab =
+      time_runs_forward
+          ? Slab::with_duration_from_start(initial_time_value,
+                                           initial_slab_size)
+          : Slab::with_duration_to_end(initial_time_value, initial_slab_size);
+  return time_runs_forward ? initial_slab.start() : initial_slab.end();
 }
 
-// Local time stepping
-template <typename Metavariables, typename DbTagsList,
-          Requires<Metavariables::local_time_stepping> = nullptr>
-TimeDelta get_initial_time_step(const Time& initial_time,
-                                const double initial_dt_value,
-                                const db::DataBox<DbTagsList>& box) {
-  const auto& step_controller = db::get<Tags::StepController>(box);
-  return step_controller.choose_step(initial_time, initial_dt_value);
+template <typename TimeStepper>
+void set_next_time_step_id(const gsl::not_null<TimeStepId*> next_time_step_id,
+                           const Time& initial_time,
+                           const bool time_runs_forward,
+                           const TimeStepper& time_stepper) {
+  *next_time_step_id = TimeStepId(
+      time_runs_forward,
+      -static_cast<int64_t>(time_stepper.number_of_past_steps()), initial_time);
 }
-}  // namespace Evolution_detail
+}  // namespace detail
 
-namespace Initialization {
-namespace Actions {
 /// \ingroup InitializationGroup
-/// \brief Initialize items related to time, such as the time step
+/// \brief Initialize items related to time stepping
 ///
-/// Since we have not started the evolution yet, we initialize the state
-/// _before_ the initial time. So `Tags::TimeStepId` is undefined at this point,
-/// and `Tags::Next<Tags::TimeStepId>` is the initial time. `::Tags::Time` is
-/// set to the initial time so it is available to dependent compute items, e.g.,
-/// Jacobians of time-dependent domains that are needed to take numerical
-/// derivatives of initial data.
+/// \details See the type aliases defined below for what items are added to the
+/// GlobalCache, MutableGlobalCache, and DataBox and how they are initialized
 ///
-/// DataBox changes:
-/// - Adds:
-///   * Tags::TimeStepId
-///   * `Tags::Next<Tags::TimeStepId>`
-///   * Tags::TimeStep
-///   * Tags::Time
-/// - Removes: nothing
-/// - Modifies: nothing
-///
-/// \note HistoryEvolvedVariables is allocated, but needs to be initialized
-template <typename Metavariables>
-struct TimeAndTimeStep {
+/// Since the evolution has not started yet, initialize the state
+/// _before_ the initial time. So `Tags::TimeStepId` is undefined at this point,
+/// and `Tags::Next<Tags::TimeStepId>` is the initial time.
+template <typename Metavariables, bool UsingLts>
```

Comment:
We could also pass in the `step_chooser_simple_tags` directly instead of the Metavariables, but I'm totally fine doing that later in order to get this merge :)

Reply:
I will do something later when I deal with step_chooser_compute_tags (which are now handled by calling AddComputeItems, but should be able to just add them to one of the initialization mutators)

Reply:
See also #4484.